code/modules/surgery/organs/liver.dm

```diff
@@ -16,6 +16,8 @@
 	var/toxTolerance = LIVER_DEFAULT_TOX_TOLERANCE//maximum amount of toxins the liver can just shrug off
 	var/toxLethality = LIVER_DEFAULT_TOX_LETHALITY//affects how much damage toxins do to the liver
 	var/filterToxins = TRUE //whether to filter toxins
+	var/swelling = 0

```

Comment:
what even causes swelling because movespeed AND clickdelay is a really, really heavy impact.

Reply:
Swelling occurs when the chem is inside of you, it's used to calculate the effects of move speed and click delay, while also providing a buffer about about 50u reagents before having any affect. The swelling var is incremented using the PE/BE reagents themselves.